<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QvisCapturePreferencesDialog</class>
 <widget class="QDialog" name="QvisCapturePreferencesDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>373</width>
    <height>378</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Capture Preferences</string>
  </property>
  <layout class="QVBoxLayout" name="MainLayout">
   <item>
    <widget class="QGroupBox" name="FileGroupBox">
     <property name="title">
      <string/>
     </property>
     <layout class="QVBoxLayout" name="FileLayout">
      <item>
       <layout class="QHBoxLayout" name="FineNameLayout">
        <item>
         <widget class="QLabel" name="FileNameLabel">
          <property name="text">
           <string>File name</string>
          </property>
          <property name="buddy">
           <cstring>FileNameText</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="FileNameText">
          <property name="text">
           <string>fluorender</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="FileNameFamilyCheckBox">
          <property name="text">
           <string>Family</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QCheckBox" name="EmbedInProjectFolderCheckBox">
        <property name="text">
         <string>Embed all files in the project folder</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="DirectoryLayout">
        <item>
         <widget class="QLabel" name="DirectoryLabel">
          <property name="text">
           <string>Directory</string>
          </property>
          <property name="buddy">
           <cstring>DirectoryText</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="DirectoryText">
          <property name="text">
           <string>.</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="DirectoryButton">
          <property name="text">
           <string>...</string>
          </property>
          <property name="autoDefault">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="FileTypeGroupBox">
     <property name="title">
      <string/>
     </property>
     <layout class="QVBoxLayout" name="FileTypeLayout">
      <item>
       <widget class="QCheckBox" name="LZWCompressionCheckBox">
        <property name="text">
         <string>LZW (Lempel-Ziv-Welch) Compression</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="SaveAlphaChannelCheckBox">
        <property name="text">
         <string>Save alpha channel</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="SaveFloatChannelCheckBox">
        <property name="text">
         <string>Save float channel</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="ScaleOutputImageGroupBox">
     <property name="title">
      <string>Scale output image</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <layout class="QHBoxLayout" name="ScaleOutputImageLayout">
      <item>
       <widget class="QSlider" name="ScaleOutputImageSlider">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>100</number>
        </property>
        <property name="value">
         <number>10</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="ScaleOutputImageSpinBox">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="decimals">
         <number>1</number>
        </property>
        <property name="minimum">
         <double>0.100000000000000</double>
        </property>
        <property name="maximum">
         <double>10.000000000000000</double>
        </property>
        <property name="value">
         <double>1.000000000000000</double>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="CancelButton">
       <property name="text">
        <string>Cancel</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="OkButton">
       <property name="text">
        <string>Ok</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>FileNameText</tabstop>
  <tabstop>FileNameFamilyCheckBox</tabstop>
  <tabstop>EmbedInProjectFolderCheckBox</tabstop>
  <tabstop>DirectoryText</tabstop>
  <tabstop>DirectoryButton</tabstop>
  <tabstop>ScaleOutputImageSlider</tabstop>
  <tabstop>ScaleOutputImageSpinBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>LZWCompressionCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>QvisCapturePreferencesDialog</receiver>
   <slot>LZWCompressionToggled  (bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>255</x>
     <y>168</y>
    </hint>
    <hint type="destinationlabel">
     <x>271</x>
     <y>49</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>SaveAlphaChannelCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>QvisCapturePreferencesDialog</receiver>
   <slot>SaveAlphaChannelToggled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>128</x>
     <y>189</y>
    </hint>
    <hint type="destinationlabel">
     <x>274</x>
     <y>83</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>SaveFloatChannelCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>QvisCapturePreferencesDialog</receiver>
   <slot>SaveFloatChannelToggled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>140</x>
     <y>210</y>
    </hint>
    <hint type="destinationlabel">
     <x>273</x>
     <y>79</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>EmbedInProjectFolderCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>QvisCapturePreferencesDialog</receiver>
   <slot>EmbedInProjectFolderToggled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>268</x>
     <y>79</y>
    </hint>
    <hint type="destinationlabel">
     <x>274</x>
     <y>187</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>DirectoryButton</sender>
   <signal>clicked()</signal>
   <receiver>QvisCapturePreferencesDialog</receiver>
   <slot>DirectoryButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>344</x>
     <y>111</y>
    </hint>
    <hint type="destinationlabel">
     <x>275</x>
     <y>220</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ScaleOutputImageSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>QvisCapturePreferencesDialog</receiver>
   <slot>ScaleOutputImageSliderChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>287</x>
     <y>292</y>
    </hint>
    <hint type="destinationlabel">
     <x>274</x>
     <y>168</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ScaleOutputImageSpinBox</sender>
   <signal>valueChanged(double)</signal>
   <receiver>QvisCapturePreferencesDialog</receiver>
   <slot>ScaleOutputImageSpinBoxChanged(double)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>345</x>
     <y>290</y>
    </hint>
    <hint type="destinationlabel">
     <x>273</x>
     <y>157</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>FileNameText</sender>
   <signal>textChanged(QString)</signal>
   <receiver>QvisCapturePreferencesDialog</receiver>
   <slot>FileNameTextChanged(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>224</x>
     <y>48</y>
    </hint>
    <hint type="destinationlabel">
     <x>367</x>
     <y>212</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>FileNameFamilyCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>QvisCapturePreferencesDialog</receiver>
   <slot>FileNameFamilyToggled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>345</x>
     <y>49</y>
    </hint>
    <hint type="destinationlabel">
     <x>369</x>
     <y>198</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ScaleOutputImageGroupBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>QvisCapturePreferencesDialog</receiver>
   <slot>ScaleOutputImageToggled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>47</x>
     <y>241</y>
    </hint>
    <hint type="destinationlabel">
     <x>5</x>
     <y>225</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>DirectoryText</sender>
   <signal>returnPressed()</signal>
   <receiver>QvisCapturePreferencesDialog</receiver>
   <slot>DirectoryTextReturnPressed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>192</x>
     <y>102</y>
    </hint>
    <hint type="destinationlabel">
     <x>186</x>
     <y>181</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>OkButton</sender>
   <signal>clicked()</signal>
   <receiver>QvisCapturePreferencesDialog</receiver>
   <slot>OkButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>336</x>
     <y>335</y>
    </hint>
    <hint type="destinationlabel">
     <x>186</x>
     <y>181</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>CancelButton</sender>
   <signal>clicked()</signal>
   <receiver>QvisCapturePreferencesDialog</receiver>
   <slot>CancelButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>264</x>
     <y>335</y>
    </hint>
    <hint type="destinationlabel">
     <x>186</x>
     <y>181</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>LZWCompressionToggled  (bool)</slot>
  <slot>SaveAlphaChannelToggled(bool)</slot>
  <slot>SaveFloatChannelToggled(bool)</slot>
  <slot>ScaleOutputImageToggled(bool)</slot>
  <slot>ScaleOutputImageSliderChanged(int)</slot>
  <slot>ScaleOutputImageSpinBoxChanged(double)</slot>
  <slot>EmbedInProjectFolderToggled(bool)</slot>
  <slot>FileNameTextChanged(QString)</slot>
  <slot>FileNameFamilyToggled(bool)</slot>
  <slot>DirectoryButtonClicked()</slot>
  <slot>DirectoryTextReturnPressed()</slot>
  <slot>OkButtonClicked()</slot>
  <slot>CancelButtonClicked()</slot>
 </slots>
</ui>
